Source/sync.cpp

```diff
@@ -237,7 +236,7 @@ static void sync_monster(int pnum, const TSyncMonster *p)
 				M_ClearSquares(ndx);
 				dMonster[monster[ndx].position.tile.x][monster[ndx].position.tile.y] = ndx + 1;
 				M_WalkDir(ndx, md);
-				monster[ndx]._msquelch = UCHAR_MAX;
+				monster[ndx]._msquelch = UINT8_MAX;

```

Comment:
The C++ way of doing this is:

```c++
#include <limits>

monster[ndx]._msquelch = std::numeric_limits<std::uint8_t>::max();
```

Reply:
ok, maybe at some point when rooting out C code :)